aztec-up/bin/aztec-install

```diff
@@ -110,8 +110,19 @@ if [ -z "${SKIP_PULL:-}" ]; then
 fi
 
 # Download the Docker Compose file. Used by aztec.
-curl -fsSL http://$INSTALL_HOST/docker-compose.sandbox.yml -o $AZTEC_PATH/docker-compose.sandbox.yml
-curl -fsSL http://$INSTALL_HOST/docker-compose.test.yml -o $AZTEC_PATH/docker-compose.test.yml
+function download_docker_compose {
+  local install_url
+  if [ "$VERSION" != "latest" ]; then
+    install_url="http://$INSTALL_HOST/$VERSION/docker-compose.${1}.yml"
+  else
+    install_url="http://$INSTALL_HOST/master/docker-compose.${1}.yml"
```

Comment:
actually this should just be `install_url="http://$INSTALL_HOST/docker-compose.${1}.yml`